tests/FINDOLOGIC/Export/Tests/XmlSerializationTest.php

```diff
@@ -30,19 +35,20 @@
  */
 class XmlSerializationTest extends TestCase
 {
-    const SCHEMA_URL = 'https://raw.githubusercontent.com/findologic/xml-export/master/src/main/resources/findologic.xsd';
+    private const SCHEMA_URL = 'https://raw.githubusercontent.com/findologic/xml-export/master/src/main/resources/' .
```

Comment:
Insead of downloading the schema like this, use [findologic/xml-export-schema](https://packagist.org/packages/findologic/xml-export-schema) as a dev dependency.

Reply:
Added the class `Constant` from the project. We just need to wait for the packagist update of the dependency with version `1.2.0`.

Reply:
@mmachatschek The new version is now on Packagist.